src/vivarium_gates_mncnh/data/loader.py

```diff
@@ -428,19 +428,17 @@ def load_paf_data(
     key: str, location: str, years: Optional[Union[int, str, list[int]]]
 ) -> pd.DataFrame:
     if key == data_keys.LBWSG.PAF:
-        filename = (
-            "calculated_lbwsg_paf_on_cause.all_causes.cause_specific_mortality_rate.parquet"
-        )
+        filename = "calculated_lbwsg_paf_on_cause.all_causes.all_cause_mortality_risk.parquet"
     else:
-        filename = "calculated_lbwsg_paf_on_cause.all_causes.cause_specific_mortality_rate_preterm.parquet"
+        filename = "calculated_lbwsg_paf_on_cause.all_causes.all_cause_mortality_risk_preterm.parquet"
 
     location_mapper = {
         "Ethiopia": "ethiopia",
         "Nigeria": "nigeria",
         "Pakistan": "pakistan",
     }
 
-    output_dir = paths.PAF_DIR / location_mapper[location]
+    output_dir = paths.PAF_DIR / "temp_outputs" / location_mapper[location]
```

Comment:
I'm having difficulty writing to the PAF dir so I've made this temporary directory for now. @albrja I'd like to figure out permissions stuff so we can write to the original directory before merging this PR